layouts/article.html

```diff
@@ -15,6 +15,12 @@
         <div class="markdown-body document-content">
           <% section = lookup_section('toc', @item[:title]) %>
           <h1><%= section << ' ' << @item[:title] if section %></h1>
+          <% improve_this_page_url = "https://github.com/atom/flight-manual.atom.io/edit/master/content" + @item.identifier %>
```

Comment:
I would put the URL directly in the `href` below like this:

```
<a href="<%= "https://github.com/atom/flight-manual.atom.io/edit/master/content#{@item.identifier}" %>">
```

No need for a variable :grinning: